qiita_db/study.py

```diff
@@ -347,6 +348,56 @@ def create(cls, owner, title, efo, info, investigation=None):
 
         return cls(study_id)
 
+    @classmethod
+    def delete(cls, id_):
+        r"""Deletes the study from the database
+
+        Parameters
+        ----------
+        id_ : integer
+            The object identifier
+
+        Raises
+        ------
+        QiitaDBError
+            If the sample_(id_) table exists means a sample template exists
+        """
+        cls._check_subclass()
+
```

Comment:
Need to check if the study with id 'id_' exists.

Off topic: This is another reason why I think `delete` should be an instance method.